From bff7a6b4cc6b680b8f6be37cd1112b370a716ab1 Mon Sep 17 00:00:00 2001
From: Legimet <legimet.calc@gmail.com>
Date: Wed, 2 Jul 2014 16:31:28 -0400
Subject: [PATCH] Modify for TI-Nspire

---
 libopenjpeg/opj_malloc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libopenjpeg/opj_malloc.h b/libopenjpeg/opj_malloc.h
index b279445..378f4e0 100644
--- a/libopenjpeg/opj_malloc.h
+++ b/libopenjpeg/opj_malloc.h
@@ -88,7 +88,7 @@ Allocate memory aligned to a 16 byte boundry
   #elif defined(__FreeBSD__)
     #define HAVE_POSIX_MEMALIGN
 	/* Linux x86_64 and OSX always align allocations to 16 bytes */
-	#elif !defined(__amd64__) && !defined(__APPLE__) && !defined(_AIX)
+	#elif !defined(__amd64__) && !defined(__APPLE__) && !defined(_AIX) && !defined(_TINSPIRE)
 		#define HAVE_MEMALIGN
 		#include <malloc.h>			
 	#endif
-- 
2.0.1

